bitcoin/chainparams.c

```diff
@@ -0,0 +1,73 @@
+#include "chainparams.h"
+#include <string.h>
+
+#define CHAINPARAMS_NETWORK_COUNT 4
+
+const struct chainparams networks[CHAINPARAMS_NETWORK_COUNT] = {
```

Comment:
Thanks for the pointer, I was indeed looking for a way to determine the size of a statically allocated array :+1: